web/thyra-frontend/src/pages/Home/home.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useMemo } from "react";
+import thyraLogo from "../../assets/ThyraLogo-V0-Detailed.png";
+import massaLogoLight from "../../assets/MASSA_LIGHT_Detailed.png";
+import massaLogomark from "../../assets/massa_logomark_detailed.png";
+import { useQuery, useMutation, useQueryClient } from "react-query";
+import gearingLogo from "../../assets/gearing.png";
+import { Link, Route, Routes } from "react-router-dom";
+import Manager from "../Plugin_Manager/manager";
+type Props = {};
+
+/**
+ * Homepage of Thyra with a list of plugins installed
+ *
+ */
+function Home({}: Props) {
+    // Fetch plugins installed by calling get /plugin/manager
+
+    // List of plugins
+    let pluginList: JSX.Element[] = [<> Loading... </>];
+    const getPlugins = async () => {
+        const init = {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+            },
+        };
+        const res = await fetch("https://my.massa/thyra/plugin-manager", init);
```

Comment:
I think we should use something more modular such as:
```suggestion
        const res = await fetch(`//${window.location.hostname}/thyra/plugin-manager`, init);
```
Which should automatically check if it is using `http` or `https` and will redirect to the same hostname, for example, if Thyra is used without the DNS.

Reply:
Yeah for sure this has to be upgraded. because it was not working when i was building it . Will go through a rework after the plugin manager page